### DISCUSSION TITLE
[✨] Case insensitive route matching

### DISCUSSION THREAD

**Discussion by jessekinsman** [QwikDev/qwik-evolution#124] [Proposals For Qwik]
### Is your feature request related to a problem?

I have a need to match both the following routes
`/Hello`
and
`/hello`

Since routing relies on filesystems directories and most file system directories are not case sensitive, it makes it impossible to match routes that are the same (when all lowercase) but have uppercase and lowercase characters.

### Describe the solution you'd like

Add a configuration that would allow QwikCity routing to be case insensitive
For example:
`/hello`
and
`/Hello`
Would be matched by the same route

### Describe alternatives you've considered

We have considered doing a catchall route that will redirect if the case in the requested url does not match the routes. 
`[...catchall]`
However, if you have a need for a route that matches both upper and lowercase, this does not work as you have to change the case and redirect and we want to keep the case intact

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
